source/expectations/data_values/ut_compound_data_helper.pkb

```diff
@@ -95,37 +95,39 @@ create or replace package body ut_compound_data_helper is
       with
         expected_cols as ( select :a_expected as item_data from dual ),
         actual_cols as ( select :a_actual as item_data from dual ),
-  expected_cols_info as (
-          select e.*,
+        expected_cols_info as (
+          select /*+ cardinality(e 100) */
+                 e.*,
                  replace(expected_type,'VARCHAR2','CHAR') expected_type_compare
             from (
-                   SELECT rownum expected_pos,
-                   xt.name expected_name,
-                   xt.type expected_type
-         FROM   (select ]'||l_column_filter||q'[ from expected_cols ucd) x,
-           XMLTABLE('/ROW/*'
-             PASSING x.item_data
-             COLUMNS 
-               name     VARCHAR2(4000)  PATH '@xml_valid_name',
-               type      VARCHAR2(4000) PATH '/' 
-             ) xt
-            ) e
+                  select rownum expected_pos,
+                         xt.name expected_name,
+                         xt.type expected_type
+                    from (select ]'||l_column_filter||q'[ from expected_cols ucd) x,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: An illegal character with code point 10 was found in this literal. ([plsql:LiteralsNonPrintableCharactersCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3ALiteralsNonPrintableCharactersCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=756&issues=AWYOC5W_vOWp7Yd6ROCk&open=AWYOC5W_vOWp7Yd6ROCk)

[ISSUE_KEY]: AWYOC5W_vOWp7Yd6ROCk